File: examples/printend.py
def fib(n):
    a, b = 0, 1
    while a < n:
        print(a, end=' ')
        a, b = b, a + b
    print()


def main():
    fib(1000)
    print("All done!", end='')


if __name__ == '__main__':
    main()

File: examples/printend.go
package main

import "fmt"

func fib(n int) {
	a, b := 0, 1
	for a < int(n) {
		fmt.Print(a, " ")
		a, b = b, a+b
	}
	fmt.Println()
}

func main() {
	fib(1000)
	fmt.Print("All done!")
}
